index.html

```diff
@@ -10244,6 +10262,43 @@ <h3>State change notification</h3>
 		<h2>Definitions of States and Properties (all aria-* attributes)</h2>
 		<p>Below is an alphabetical list of <abbr title="Accessible Rich Internet Applications">WAI-ARIA</abbr> <a>states</a> and [=ARIA/properties=] to be used by rich internet application authors. A detailed definition of each <abbr title="Accessible Rich Internet Applications">WAI-ARIA</abbr> state and [=ARIA/property=] follows this compact list.</p>
 		<p id="index_state_prop" class="placeholder">Placeholder for index of states and properties</p>
+		<div class="property" id="aria-actions">
+			<pdef>aria-actions</pdef>
+			<div class="property-description">
+				<p><a>Identifies</a> a related element or elements whose primary activation (<code>click</code> event) will trigger a behavior or operation relevant to the referencing user interface object, such as the close button for a dialog, or a reply button relevant to the email message that references it.</p>
+				<p>The <code>aria-actions</code> attribute allows an element to reference other interactive elements which trigger actions on the current element. For example, in a web mail application, the currently selected email message could reference action elements such as the <em>message-specific</em> Reply, Forward, and Delete buttons. Actions are triggered by mainstream pointer events (click, tap) or by simulated pointer events from assistive technology.</p>
```

Comment:
The term "mainstream pointer events" seems problematic to me. It is a potential point of confusion. What is a non-mainstream pointer event? I don't find this term anywhere else in the spec. I see a couple of other uses of the word "mainstream", and in all cases, I think the spec would be better without it.

I am also confused by the term "current element". What makes an element "current"? We have the property "aria-current", and using the term "current element" in this context could lead to misinterpretation of the definition of aria-actions.

I suggest:

```suggestion
				<p>The <code>aria-actions</code> attribute allows an element to reference other interactive elements which trigger actions on the referencing element. For example, in a web mail application, the selected email message could reference buttons that perform actions on that email, such as Reply, Forward, and Delete. Actions are triggered by pointer events (click, tap) that can be simulated by assistive technologies.</p>
```